src/transition.js

```diff
@@ -76,13 +77,11 @@ export default class Transition {
   }
 
   _startTransitionTimer() {
-    var that = this;
-
     if (this.transitionIntervalId === null) {
-      that._startTransition();
-      this.transitionIntervalId = setInterval(function() {
-        that._startTransition();
+      this.transitionIntervalId = setInterval( () => {
+        this._startTransition();
       }, this.intervalTime);
+      this._startTransition();
```

Comment:
Any particular reason why this was moved?

Reply:
@jungeunyoon yes, when there is only one tweet it would call finalize before starting the setInterval. cheers